docs/content/concepts/ops-jobs-graphs/metadata-tags.mdx

```diff
@@ -1,25 +1,34 @@
 ---
-title: Job Metadata & Run Tags | Dagster
-description: Job metadata and run tags provide two ways to attach information in Dagster
+title: "Op job metadata | Dagster Docs"
+description: Attach information to op jobs using metadata
 ---
 
-# Job Metadata & Run Tags
+# Op job metadata
 
-Metadata and tags provide two different ways for you to attach information to your jobs and the runs launched from those jobs. The main difference between metadata and tags is what "object" the information is attached to. Metadata is attached to JobDefinitions (specified using the <PyObject object="job" decorator /> decorator), and tags are attached to runs that are created by executing a job.
```

Comment:
> The main difference between metadata and tags is what "object" the information is attached to

I think the way that we want to frame the main difference between metadata and tags is now different than when this was written.

Tags are intended to be used for organization. They help you find the objects that they're attached to. They can be used to filter search results and select subsets. They are expected to be relatively short.

Metadata is intended to be used once you've already found the object that it's on. Metadata can be large.